src/main/java/com/rainbowrave/Curve.java

```diff
@@ -0,0 +1,37 @@
+package com.rainbowrave;
+
+import java.util.ArrayList;
+import net.runelite.api.Point;
+
+public class Curve {
+
+    private final Point from;
+    private final Point to;
+    private final ArrayList<Point> curve = new ArrayList<>();
```

Comment:
Variable types should be List not the List implementation ArrayList.

Reply:
Thanks, I knew I was forgetting something basic about my declarations.